### ISSUE TITLE
Do we still need localUserID memory test?

### ISSUE THREAD

**Issue by jcampbell05** [Supmenow/sup-ios-issues#160]
If the real issue about the localUserID bug is first time logins, then do we still need the memory and disk tests ?

We should look if if `NSUserDefaults` is enough based on logs and testing. Then simplify the code to just use that.

**Comment by roger-tan** [comment]
We need to investigate in couple days when we have enough feedbacks

**Comment by jcampbell05** [comment]
Agreed, will keep an eye on LogEntries, if most of them are successfully loading from `NSUserDefaults` then we can simplify.
